persistence/db.go

```diff
@@ -180,6 +180,10 @@ func initializeGovTables(ctx context.Context, db *pgxpool.Conn) error {
 		return err
 	}
 
+	if _, err := db.Exec(ctx, types.UpgradesTableSchema); err != nil {
```

Comment:
> Does this work as expected?

yes

> Ideally it should be consistent with the above db.Exec statements.

why? i don't want to get into an argument about how to prepare sql statements, but i am strongly against this pattern of dynamically building our own sql statements. IMO it's a waste of time, reduces clarity into what is actually in a transaction, and typically gives little benefit at the guise of being DRY. I would be in favor of introducing a SQL builder library instead of handwriting our sql but even that feels like overkill and not needed.